utils/get_latest_copr_build

```diff
@@ -1,28 +1,55 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
```

Comment:
Actually I tried to make it work also for python2 because you can install Copr-cli via pip for python2
Which kind of helps if you use a rhel7

Reply:
Hmm. I will rather create new env var for Python2 to be used if specified from the cmdline. But Python3 should be default nowadays.